packages/react/src/constants/CardPropTypes.js

```diff
@@ -230,6 +230,8 @@ export const TableCardPropTypes = {
   cardVariables: PropTypes.objectOf(
     PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.number, PropTypes.bool])
   ),
+  /** Display tooltips in the table toolbar (will reset overflow property) */
+  withToolbarTooltips: PropTypes.bool,
```

Comment:
is this something that can be specific to the TableCard?

Reply:
This prop is part of specific `TableCardPropTypes` or you mean to move it inside `TableCard` component?

Reply:
Yes, inside the TableCard component

Reply:
Got it, I will fix it.